kong-2.8.0-0.rockspec

```diff
@@ -41,6 +41,7 @@ dependencies = {
   "lua-resty-ipmatcher == 0.6.1",
   "lua-resty-acme == 0.8.0",
   "lua-resty-session == 3.10",
+  "luarocks == 3.9.0",
```

Comment:
Do I need to do this? Or is luarocks a default dependency and we shouldn't pin it here so we don't end up with 2 installed versions if the included version is updated later?

Reply:
No, this is not needed. It may cause trouble down the line too if versions are updated elsewhere.